pkgs/by-name/az/azure-cli/extensions-manual.nix

```diff
@@ -42,6 +42,20 @@
     ];
     meta.maintainers = with lib.maintainers; [ giggio ];
   };
+  ssh = mkAzExtension rec {
```

Comment:
We try to keep this sorted alphabetically, so this should go underneath rdbms-connect.